src/api-reference/request/get-started.markdown

```diff
@@ -0,0 +1,173 @@
+  <img src="https://run.pstmn.io/button.svg" alt="Run in Postman">
+</a>
+
+#### <a name="token"></a>Retrieve a User Access Token:
+
+Before making requests to the Requests API, you must [obtain an access token from the Authentication API](https://developer.concur.com/api-reference/authentication/getting-started.html).
+
+The response will include an `access_token` field, which contains your access token. For subsequent calls, you will need to include this access token in the `Authorization` header of your calls. An `id_token` will be also included in the response. In order to retrieve the unique ID for your user, you will have to decode this `id_token` at [jwt.io](https://jwt.io/). You will need this ID in order to post requests.
+
+Examples:
+
+cURL:
+
+```shell
+curl -d "client_secret={YOUR SECRET}&client_id={YOUR CLIENT ID}&grant_type=password&username={YOUR USERNAME}&password={YOUR PASSWORD}" https://us.api.concursolutions.com/oauth2/v0/token
+```
+
+HTTPie:
+
+```shell
+http -f POST https://us.api.concursolutions.com/oauth2/v0/token client_secret={YOUR SECRET} client_id={YOUR CLIENT ID} grant_type=password username={YOUR USERNAME} password=P{YOUR PASSWORD}
+```
+
+#### <a name="javascript">Explore the API in JavaScript
+
+Below are some simple NodeJS code snippets for getting a token and posting a request.
+
+##### <a name="retrieve-token"></a>Retrieve a User Access Token:
+
+```js
+'use strict';
+const request = require('request');
+
+request.post({
+    url:'https://us.api.concursolutions.com/oauth2/v0/token',
+    form: {
+        client_secret: 'YOUR VALUE HERE',
+        client_id: 'YOUR VALUE HERE',
+        username: 'YOUR VALUE HERE',
+        password: 'YOUR VALUE HERE',
+        grant_type: 'password'
+    }},
+    (err, httpResponse, body) => {
+        if(err) { console.log(err); }
+        console.log('Status:', httpResponse.statusCode);
+        console.log('Response:', body);
+    });
+```
+
+##### <a name="post-request"></a>Post a Request
+
+```js
+'use strict';
+const https = require('https');
+
+const ACCESS_TOKEN = 'YOUR ACCESS TOKEN HERE';
+const USER_ID = 'YOUR VALUE HERE';
+const request = JSON.stringify(YOUR_REQUEST_HERE);
+
+const options = {
+    hostname: 'us.api.concursolutions.com',
+    path: `/travelrequest/v4/users/${USER_ID}`,
+    method: 'POST',
+    headers: {
+        'Authorization': `Bearer ${ACCESS_TOKEN}`,
+        'Content-Type': 'application/json',
+        'Content-Length': Buffer.byteLength(request),
+        'Link': '<https://schema.concursolutions.com/general-receipt.schema.json>;rel=describedBy'
+    }
+};
+
+const req = https.request(options, (res) => {
+    console.log('statusCode:', res.statusCode);
+    console.log('headers:', res.headers);
+
+    res.on('data', (data) => {
+        process.stdout.write(data);
+    });
+});
+
+req.write(receipt);
```

Comment:
_note: update valeur (receipt to Request) and perform tests to make sure this works as described._